### PR TITLE
Update .gitignore

### PR DESCRIPTION
**Pull request by mohsenuss91**
.gitignore for Python and some popular IDE added.

### CODE REVIEW

**Comment by CodeClubRik** [comment]
Thanks for this.

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,140 @@
...
+.coverage.*
+.cache
+nosetests.xml
+coverage.xml
+*,cover
+.hypothesis/
+
+# Translations
+*.mo
+*.pot
```
Comment (kernc) [inline .gitignore]: Nice .gitignore. I think .pot templates, albeit generated, usually are version tracked.

Comment (andylolz) [inline .gitignore]: This .gitignore derives from https://github.com/github/gitignore

Comment (mohsenuss91) [inline .gitignore]: Actually it's from  www.gitignore.io

Comment (andylolz) [inline .gitignore]: Ah, gotcha. But https://github.com/github/gitignore is the source for that site:
https://github.com/joeblau/gitignore.io/tree/master/data

**Pull request closed without merging**